Assignment1/index.html

```diff
@@ -1,35 +1,52 @@
-<DOCTYPE html>
+<!DOCTYPE html>
 <html>
     <head>
          <style>
+<<<<<<< HEAD
```

Comment:
This are the git conflicts and should be remove manually. You need to choose which one to keep